MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -663,7 +663,13 @@ internal void CreateSizeDependentResources()
                                             PresentationParameters.BackBufferWidth,
                                             PresentationParameters.BackBufferHeight,
                                             format,
-                                            SwapChainFlags.None);
+                                            SwapChainFlags.None
+                                            );
+
+                if (useFullscreenParameter)
```

Comment:
Can you add a comment here as to what this is doing?  Like I don't understand why we need `useFullscreenParameter` and don't just obey the `PresentationParameters.IsFullScreen` value.